src/ServiceFabric/ServiceFabric/ServiceFabric.csproj

```diff
@@ -16,4 +16,11 @@
     <PackageReference Include="Microsoft.Azure.Management.ServiceFabric" Version="1.1.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Template\Linux\parameter.json" CopyToOutputDirectory="PreserveNewest" />
+    <None Update="Template\Linux\template.json" CopyToOutputDirectory="PreserveNewest" />
+    <None Update="Template\Windows\parameter.json" CopyToOutputDirectory="PreserveNewest" />
+    <None Update="Template\Windows\template.json" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Please replace these 4 lines with this instead.
```xml
<None Update="Template\**\*.json" CopyToOutputDirectory="PreserveNewest" />
```

Reply:
done, thanks.